ml-agents/mlagents/trainers/ppo/models.py

```diff
@@ -109,6 +109,7 @@ def create_cc_actor_critic(
             self.act_size[0],
             activation=None,
             kernel_initializer=LearningModel.scaled_init(0.01),
+            reuse=tf.AUTO_REUSE
```

Comment:
Missing comma here is what's (currently) breaking CI

Reply:
Thanks -- fixed